storybookjs/storybook
# Blocks: Redo stories for controls

Issue: NA

This PR touches on all the stories for Control components in `@storybook/blocks`. Most of them have been refactored to use a leaner CSF3 format, many of them have gotten more stories covering more use cases, while a few have gotten completely new stories.

You could argue that many of the stories would benefit from play functions, but I think that's out of scope for now.

## How to test

- See [the Storybook](https://635781f3500dd2c49e189caf-ulmxgkrttf.chromatic.com/?path=/docs/storybook-blocks-controls-boolean--docs)
- See [the changes in Chromatic](https://www.chromatic.com/pullrequest?appId=635781f3500dd2c49e189caf&number=19744) @tmeasday you might even want to review the changes and approve them.
- See the new structure for stories here:
![image](https://user-images.githubusercontent.com/5678122/199812995-a3fef646-27db-408a-9e9e-81b4b5e01a47.png)
![image](https://user-images.githubusercontent.com/5678122/199813024-0c075a18-8b1c-47a7-b87e-e5f66497aece.png)
